src/components/Calendar/CalendarPaneToolbar.js

```diff
@@ -64,19 +66,26 @@ const CalendarPaneToolbar = (props) => {
         setAnchorEl(null);
     };
 
+    const handleScheduleClick = () => {
+        setOpenSchedules((prev) => !prev);
+    };
+
     return (
         <Paper elevation={0} variant="outlined" square className={classes.toolbar}>
-            <AddScheduleDialog />
+            <EditSchedule scheduleNames={props.scheduleNames} scheduleIndex={props.currentScheduleIndex} />
```

Comment:
Deference these from props at the top of the function.

```jsx
const { classes, scheduleNames, currentScheduleIndex } = props;
```

```suggestion
            <EditSchedule scheduleNames={scheduleNames} scheduleIndex={currentScheduleIndex} />
```